### PR TITLE
(feat): Chaos Experiment: Added disk loss chaos experiment logic and job for AWS

### PR DESCRIPTION
**Pull request by rajdas98**
Signed-off-by: Raj <mail.rajdas@gmail.com>

**What this PR does / why we need it**:
- [x] Added chaos experiment logic for disk loss
- [x] Added aws configuration and dependencies
- [x] Added k8s jobs for disk loss
- [x] Logs for this experiment are mentioned here 
[aws-logs.txt](https://github.com/litmuschaos/litmus/files/3803959/aws-logs.txt)

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Checklist**

* [x] Does this PR have a corresponding GitHub issue?
https://github.com/litmuschaos/litmus/issues/906
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [ ] Have you added debug messages where necessary? 
* [x] Have you added task comments where necessary? 
* [x] Have you tested the changes for possible failure conditions?
* [x] Have you provided the positive & negative test logs for the litmusbook execution?
* [x] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [x] Have you used non-shell/command modules for Kubernetes tasks?
* [x] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [x] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [x] Do the artifacts follow the appropriate directory structure? 
* [x] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [x] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [x] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [x] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [x] Can you suggest the minimal resource requirements for the litmusbook execution?
* [x] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [x] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

File: experiments/generic/disk_loss/disk_loss_ansible_logic.yml [hunk 1]
```diff
@@ -72,7 +83,12 @@
         ## POST-CHAOS DISK LIVENESS CHECK
         - name: Verify that the disk is connected to node (post)
           include_tasks: "/utils/cloud/gcp/status_disk.yml"
-
+          when: "cloud_platform == 'GCP'"
+        
+        - name: Verify that the disk is connected to node (pre)
```
Comment (shubhamchaudhary) [inline experiments/generic/disk_loss/disk_loss_ansible_logic.yml]: either remove the (pre)  or change it to (post)

**Comment by rajdas98** [comment]
Changes are done as per @ksatchit

File: experiments/generic/disk_loss/disk_loss_ansible_logic.yml [hunk 2]
```diff
@@ -47,10 +48,20 @@
...
         - name: Verify that the disk is connected to node (pre)
           include_tasks: "/utils/cloud/gcp/status_disk.yml"
+          when: "cloud_platform == 'GCP'"
+
+        - name: Verify that the disk is connected to node (pre)
+          include_tasks: "/utils/cloud/aws/status_disk.yml"
+          when: "cloud_platform == 'AWS'"
 
         # Checking disk is attached to node
         - debug:
```
Comment (uditgaurav) [inline experiments/generic/disk_loss/disk_loss_ansible_logic.yml]: what debug?

File: experiments/generic/disk_loss/disk_loss_ansible_logic.yml [hunk 3]
```diff
@@ -72,7 +83,12 @@
...
-
+          when: "cloud_platform == 'GCP'"
+        
+        - name: Verify that the disk is connected to node (post)
+          include_tasks: "/utils/cloud/aws/status_disk.yml"
+          when: "cloud_platform == 'AWS'"
+          
         # Checking disk is attached to node
         - debug: 
             msg: echo "disk attached"
```
Comment (uditgaurav) [inline experiments/generic/disk_loss/disk_loss_ansible_logic.yml]: why echo ?

File: experiments/generic/disk_loss/disk_loss_ansible_logic.yml (hunk 2 above)
Comment (rajdas98) [reply experiments/generic/disk_loss/disk_loss_ansible_logic.yml]: As @ksatchit . He told to use debug module

File: experiments/generic/disk_loss/disk_loss_ansible_logic.yml (hunk 3 above)
Comment (rajdas98) [reply experiments/generic/disk_loss/disk_loss_ansible_logic.yml]: Thanks, @uditgaurav to point out this thing

Comment (rajdas98) [reply experiments/generic/disk_loss/disk_loss_ansible_logic.yml]: Done

**Pull request merged**